HealthMonitoring.Integration.PushClient/HealthMonitorPushClient.cs

```diff
@@ -51,18 +52,29 @@ public HealthMonitorPushClient WithHealthCheck(IHealthChecker healthChecker)
             return this;
         }
 
+        public HealthMonitorPushClient WithBackOffStategy(IBackOffStategy backOffStategy)
+        {
+            _backOffStategy = backOffStategy;
```

Comment:
should our clients be bothered with our logs and their backoff strategy ? i think it should be moved out from fluent interface